nav2_bringup/launch/system_test.yaml

```diff
@@ -0,0 +1,4 @@
+map_server:
+    ros__parameters:
+        yaml_filename: "test_map.yaml"
```

Comment:
I think that for the map server, you could directly load at launch the "test_map.yaml" and use the variables as parameters within the code. We could then modify the LoadMapInfoFromFile implementation to grab these parameters instead of parsing the yaml file within the code. This could eliminate the dependency on yaml_vendor in the map_server.

Reply:
Yeah, good idea. I'll make those changes.